app/controllers/webhooks_controller.rb

```diff
@@ -69,7 +69,11 @@ def email_params
   end
 
   def authorized?
-    params[:auth].present? && params[:auth].secure_compare(ENV['WEBHOOK_AUTH_KEY'])
+    params[:auth].present? &&
+      ActiveSupport::SecurityUtils.secure_compare(
+        params[:auth],
```

Comment:
I think the problem is that this returns quicker when there's a length mismatch, which allows an attacker to determine the key length.